src/test/java/bio/terra/common/logging/LoggingTest.java

```diff
@@ -200,6 +201,34 @@ public void testStructuredLogging(CapturedOutput capturedOutput) {
     assertThat((String) readJson(event4, "$.foo.bar")).isEqualTo("baz");
   }
 
+  @Test
+  public void testAlertLogging(CapturedOutput capturedOutput) {
```

Comment:
have never seen test methods with parameters. til

Reply:
but also I really wonder what is this testing? is it testing the loggingTestController or loggingUtils..

Reply:
It's a bit tricky because this is a logging library - we have a toy Spring service using the library (`loggingTestController`), and this test is verifying the messages the toy service logs.

Reply:
i see. thanks for explaining